model-zoo/src/main/java/ai/djl/basicmodelzoo/BasicModelZoo.java

```diff
@@ -31,23 +27,15 @@ public class BasicModelZoo extends ModelZoo {
     private static final Repository REPOSITORY = Repository.newInstance("zoo", REPO_URL);
     public static final String GROUP_ID = "ai.djl.zoo";
 
-    private static final List<ModelLoader> MODEL_LOADERS = new ArrayList<>();
-
-    static {
+    BasicModelZoo() {
```

Comment:
NIt: Should we keep this consistent with how you are adding models in the other modelzoos? 

this
```
addModel(REPOSITORY.model(...))
```
instead of
```
Model m = REPOSITORY.model(...)
addModel(m)
```